cmd/explaintest/t/cte.test

```diff
@@ -256,3 +256,22 @@ select * from t1 where c1 > all(with recursive cte1 as (select c1, c2 from t2 un
 
 explain select * from t1 where exists(with recursive cte1 as (select c1, c2 from t2 union all select c1+1 as c1, c2+1 as c2 from cte1 where cte1.c2=t1.c2) select c1 from cte1);
 select * from t1 where exists(with recursive cte1 as (select c1, c2 from t2 union all select c1+1 as c1, c2+1 as c2 from cte1 where cte1.c2=t1.c2) select c1 from cte1);
+--echo // Test CTE name is the same as table name
+use test;
+drop table if exists t1, t2;
+drop view if exists v1;
+create table t1 (a int);
+insert into t1 values (0), (1), (2), (3), (4);
+create table t2 (a int);
+insert into t2 values (1), (2), (3), (4), (5);
+with t1 as (select a from t2 where t2.a=3 union select t2.a+1 from t1,t2 where t1.a=t2.a) select * from t1 order by a desc;
+with recursive t1 as ( select a from t2 where t2.a=3 union select t2.a+1 from t1,t2 where t1.a=t2.a) select * from t1 order by a desc;
```

Comment:
Where is the `create view` statements?

Reply:
My forget. I will add it later. Now I find some cases not covered.I will think about to solve and fix it. And then i will add more cases into the `cte.test`